<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="2" tests="4" time="23.154">
 <testsuite name="Protein Search testing" timestamp="2019-05-08T12:13:30" hostname="localhost" time="23.154" errors="0" tests="4" skipped="0" disabled="0" failures="2">
  <testcase classname="Protein Search testing" name="should go to Protein Search page" time="7.359" />
  <testcase classname="Protein Search testing" name="should go to Protein list page" time="11.279" />
  <testcase classname="Protein Search testing" name="number of results found" time="2.053">
   <failure type="toContain" message="Expected &apos;&quot;3 Proteins were found&quot;&apos; to contain &apos;4&apos;."><![CDATA[Error: Failed expectation
    at browser.executeScript.then (/Applications/MAMP/htdocs/glygen-frontend/protractor_test/spec.js:51:30)
    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
  <testcase classname="Protein Search testing" name="number of records displayed per page should match the selection" time="2.46">
   <failure type="toEqual" message="Expected 3 to equal 4."><![CDATA[Error: Failed expectation
    at /Applications/MAMP/htdocs/glygen-frontend/protractor_test/spec.js:62:22
    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>